src/equations/compressible_euler_1d.jl

```diff
@@ -644,13 +648,13 @@ end
     M = v1 / a
 
     P = 2
-    mu = 1.0
-    nu = 0.75
-    omega = 2.0 # adjusted from suggested value of 1.5
+    mu = 1
+    nu = 0.75f0
```

Comment:
Is this really `0.75f0 == 0.75`? We need to write this down somewhere for future me 😅

Reply:
```julia
julia> 0.75f0 == 0.75
true
```

Reply:
🤷‍♂️ /r